[#268] Google/Facebook source documentation (#301)
airyhq/airy: docs/docs/api/http.md

This action is idempotent, so if the channel is already connected the status
will be `202`.

> Connecting a channel is source specific, so you must read the source documentation for the correct payload.

Reviewer: ```suggestion
Connecting a channel is source specific by nature, refer to the relevant documentation for the correct payload:
```
